apis/v1alpha1/types.go

```diff
@@ -27,3 +27,142 @@ var (
+}
+
+// This structure is returned in the response object of ListCertificates action.
+type CertificateSummary struct {
+	CertificateARN                       *string      `json:"certificateARN,omitempty"`
+	CreatedAt                            *metav1.Time `json:"createdAt,omitempty"`
+	DomainName                           *string      `json:"domainName,omitempty"`
+	Exported                             *bool        `json:"exported,omitempty"`
+	ExtendedKeyUsages                    []*string    `json:"extendedKeyUsages,omitempty"`
+	HasAdditionalSubjectAlternativeNames *bool        `json:"hasAdditionalSubjectAlternativeNames,omitempty"`
+	ImportedAt                           *metav1.Time `json:"importedAt,omitempty"`
+	InUse                                *bool        `json:"inUse,omitempty"`
+	IssuedAt                             *metav1.Time `json:"issuedAt,omitempty"`
+	KeyAlgorithm                         *string      `json:"keyAlgorithm,omitempty"`
+	KeyUsages                            []*string    `json:"keyUsages,omitempty"`
+	NotAfter                             *metav1.Time `json:"notAfter,omitempty"`
+	NotBefore                            *metav1.Time `json:"notBefore,omitempty"`
+	RenewalEligibility                   *string      `json:"renewalEligibility,omitempty"`
+	RevokedAt                            *metav1.Time `json:"revokedAt,omitempty"`
+	Status                               *string      `json:"status,omitempty"`
+	SubjectAlternativeNameSummaries      []*string    `json:"subjectAlternativeNameSummaries,omitempty"`
+	Type                                 *string      `json:"type_,omitempty"`
+}
+
+// Contains information about the validation of each domain name in the certificate.
+type DomainValidation struct {
+	DomainName *string `json:"domainName,omitempty"`
+	// Contains a DNS record value that you can use to validate ownership or control
+	// of a domain. This is used by the DescribeCertificate action.
+	ResourceRecord   *ResourceRecord `json:"resourceRecord,omitempty"`
+	ValidationDomain *string         `json:"validationDomain,omitempty"`
+	ValidationEmails []*string       `json:"validationEmails,omitempty"`
+	ValidationMethod *string         `json:"validationMethod,omitempty"`
+	ValidationStatus *string         `json:"validationStatus,omitempty"`
+}
+
+// Contains information about the domain names that you want ACM to use to send
+// you emails that enable you to validate domain ownership.
+type DomainValidationOption struct {
+	DomainName       *string `json:"domainName,omitempty"`
+	ValidationDomain *string `json:"validationDomain,omitempty"`
+}
+
+// The Extended Key Usage X.509 v3 extension defines one or more purposes for
+// which the public key can be used. This is in addition to or in place of the
+// basic purposes specified by the Key Usage extension.
+type ExtendedKeyUsage struct {
+	Name *string `json:"name,omitempty"`
+	OID  *string `json:"oID,omitempty"`
```

Comment:
The json encoded name for this field is improperly capitalised

Reply:
yeah, looks like an initialism fail. will correct.